app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    command, old_file, new_file = command.split(" ")
```

Comment:
Validate command
You code will not work in this cases:
command = 
""
"mv"
"mv file1.txt"
"mv file1.txt file2.txt file3.txt"